### ISSUE TITLE
[FEATURE] Cluster Metrics Monitor -  Add _cat/shards (Cluster Monitor) and  _cat/indices

### ISSUE THREAD

**Issue by brijos** [opensearch-project/alerting#611]
**Is your feature request related to a problem?**
Cluster health metrics is a helpful feature. I want to improve operational uptime. It would be great if cluster health metrics can be extended to support _cat/shards and _cat/indices as that would provide a better picture of my deployment.

**What alternatives have you considered?**
Grabbing data from performance analyzer

**Do you have any additional context?**
https://opensearch.org/docs/latest/monitoring-plugins/alerting/monitors/#create-cluster-metrics-monitor

**Comment by praveensameneni** [comment]
We plan to add this into 2.5

**Comment by AWSHurneyt** [comment]
For reference, these are example outputs from those 2 API.
[GET _cat/indices](https://opensearch.org/docs/latest/api-reference/cat/cat-indices/)
```
health status index uuid pri rep docs.count docs.deleted store.size pri.store.size
yellow open test-logs1 So2txS5yQ8WSlsejZnOC4A 1 1 10 0 6.1kb 6.1kb
yellow open test-logs0 8gLivpwIS5eo9r-_Y88rOQ 1 1 11 0  17kb  17kb
yellow open test-logs3 nC8ghA7yQzmyD942YeE5hw 1 1 10 0 6.1kb 6.1kb
yellow open test-logs2 MU5wWj7_StuzCd1MsuMZrQ 1 1 10 0 6.1kb 6.1kb
green  open .kibana_1  B5JEspGFQCC4QUFErWc10w 1 0  1 0 5.1kb 5.1kb
```
[GET _cat/shards](https://opensearch.org/docs/latest/api-reference/cat/cat-shards/)
```
index shard prirep state docs store ip node
test-logs0 0 p STARTED    11 17.2kb 127.0.0.1 integTest-0
test-logs0 0 r UNASSIGNED                     
test-logs3 0 p STARTED    10  6.2kb 127.0.0.1 integTest-0
test-logs3 0 r UNASSIGNED                     
test-logs1 0 p STARTED    10  6.2kb 127.0.0.1 integTest-0
test-logs1 0 r UNASSIGNED                     
.kibana_1  0 p STARTED     1  5.2kb 127.0.0.1 integTest-0
test-logs2 0 p STARTED    10  6.2kb 127.0.0.1 integTest-0
test-logs2 0 r UNASSIGNED

**Comment by AWSHurneyt** [comment]
This enhancement was released with v2.9.

**Comment by pajuric** [comment]
Just a heads up - it looks like this blog issue was opened in the wrong repo. Please make sure you're opening blogs in the website-project repo going forward, so they can be tracked.
